File: codewars/cpp/calculate_bmi.cpp
// https://www.codewars.com/kata/57a429e253ba3381850000fb/train/cpp


#include <iostream>
#include <cmath>

std::string bmi(double w, double h) {
    float bmi = w / pow(h, 2);

    if(bmi <= 18.5) {
        return "Underweight";
    } else if(bmi > 18.5 && bmi <= 25.0) {
        return "Normal";
    } else if(bmi > 25.0 && bmi <= 30.0) {
        return "Overweight";
    } else {
        return "Obese";
    }

    return 0;
}


int main() {
    double w, h;

    std::cin >> w >> h;
    std::cout << bmi(w, h) << "\n";

    return 0;
}

File: codewars/python/CalculateBMI.py
# Calculate BMI


def bmi(weight, height):
    bmi = (weight / pow(height, 2))

    if (bmi <= 18.5):
        return "Underweight"
    elif (bmi <= 25.0):
        return "Normal"
    elif (bmi <= 30.0):
        return "Overweight"
    else:
        return "Obese"


def main():
    weight, height = [float(x) for x in input().split()]

    print(bmi(weight, height))


if __name__ == '__main__':
    main()
